vendor/engines/saas/app/jobs/login_private_job.rb

```diff
@@ -0,0 +1,16 @@
+class LoginPrivateJob < ActiveJob::Base
+
+  def perform(params)
+    data = {
+      user: params['user'],
```

Comment:
Before we set the `user`, we should pull the primary email out of `emails` and set `user.email` if it's not already populated. It might also be useful to add an `email_verified` key with the `verified` value from the primary email.

This should apply both to `data` and to what's set on `current_user` below.